embassy-rp/src/clocks.rs

```diff
@@ -94,6 +94,7 @@ impl ClockConfig {
                     post_div1: 6,
                     post_div2: 5,
                 }),
+                delay_multiplier: 1,
```

Comment:
perhaps we should default to a higher multiplier so that it works on all boards out of the box? so the user can lower it explicitly if they know their board's xosc stabilizes faster and they care about boot speed

Reply:
Check f69e845 - made it 128